django/db/models/fields/json.py

```diff
@@ -0,0 +1,490 @@
+import json
+
+from django import forms
+from django.core import checks, exceptions
+from django.db import connection as default_connection
```

Comment:
Is there any reason to add this alias? Please use `connection`.

Reply:
I wanted to avoid having the same identifier for the connection parameters. Not an issue anymore since I imported `connections`.